rapids/rapids.sh

```diff
@@ -30,7 +30,7 @@ readonly RUN_WORKER_ON_MASTER=$(get_metadata_attribute 'dask-cuda-worker-on-mast
 readonly DEFAULT_CUDA_VERSION="10.2"
 readonly CUDA_VERSION=$(get_metadata_attribute 'cuda-version' ${DEFAULT_CUDA_VERSION})
 readonly CUDF_VERSION=$(get_metadata_attribute 'cudf-version' ${DEFAULT_CUDF_VERSION})
-readonly RAPIDS_VERSION=$(get_metadata_attribute 'rapids-version' '0.15')
+readonly RAPIDS_VERSION=$(get_metadata_attribute 'rapids-version' '0.17')
```

Comment:
Why don't use ${DEFAULT_SPARK_RAPIDS_VERSION} as the default?

Reply:
RAPIDS has a different versioning than Spark RAPIDS